spec/integration/create_a_document_spec.rb

```diff
@@ -12,45 +13,66 @@
     and_the_preview_creation_succeeded
   end
 
+  scenario "User create a document and the chosen base path is already in use" do
+    pending("@alex-ju looking into capybara tests")
+    when_i_click_on_create_a_document
+    and_i_choose_news
+    and_i_choose_a_press_release
+    and_i_fill_in_the_title_but_base_path_is_already_in_use
+    then_i_see_a_path_is_taken_error
+  end
+
   def when_i_click_on_create_a_document
     visit "/"
     click_on "New document"
   end
 
   def and_i_choose_news
-    choose "News"
+    choose "News", allow_label_click: true
     click_on "Continue"
   end
 
   def and_i_choose_a_press_release
-    choose "Press release"
+    choose "Press release", allow_label_click: true
     click_on "Continue"
   end
 
   def and_i_fill_in_the_form_fields
     fill_in "document[title]", with: "A great title"
-    fill_in "document[base_path]", with: "/government/foo"
+    @lookup_base_path_request = publishing_api_has_lookups("/news/a-great-title": nil)
+    page.evaluate_script('document.addEventListener("DOMContentLoaded", function(event)  {document.getElementById("document-title-id").blur()})')
 
     @request = stub_publishing_api_put_content(Document.last.content_id,
                                                hash_including(title: "A great title",
-                                                              base_path: "/government/foo"))
-
+                                                              base_path: "/news/a-great-title"))
+    expect(page).to have_content("/news/a-great-title")
     click_on "Save"
   end
 
   def then_i_see_the_document_exists
     expect(Document.last.title).to eq "A great title"
     expect(page).to have_content "press_release"
     expect(page).to have_content "A great title"
-    expect(page).to have_content "/government/foo"
+    expect(page).to have_content "/news/a-great-title"
   end
 
   def and_the_preview_creation_succeeded
     expect(@request).to have_been_requested
+    expect(@lookup_base_path_request).to have_been_requested
     expect(page).to have_content "Preview creation successful"
 
     expect(a_request(:put, /content/).with { |req|
       expect(req.body).to be_valid_against_schema("news_article")
     }).to have_been_requested
   end
+
+  def and_i_fill_in_the_title_but_base_path_is_already_in_use
```

Comment:
I agree with @tijmenb this is really a separate feature - could we put this in a different file? (The idea was to try and have one feature per file, to keep them short.)

Reply:
Now extracted into its own file, but currently disabled because of the aforementioned JS issues.